lib/translation_io/config.rb

```diff
@@ -99,5 +116,26 @@ def source_files_for_formats(formats)
     def to_s
       "API Key: #{api_key} | Languages: #{source_locale} => [#{target_locales.join(', ')}]"
     end
+
+    def change_domain(domain=TEXT_DOMAIN)
+      self.text_domain = domain
+      if multi_domain
+        idx = domain_names.index(domain)
+        self.api_key = domain_api_keys[idx]
+        self.source_locale = domain_source_locales[idx]
+        self.target_locales = domain_target_locales[idx]
+        self.forced_source_paths = domain_folders[idx]
+        avoid_folders = (domain_folders - [forced_source_paths]).flatten
+        self.ignored_source_paths = ['vendor/', 'tmp/'] + avoid_folders
+        # dont need to do anything if the folder is none
+        # CASE NONE: want to ignore all folders defined in other domains
+        # CASE Folder: TODO How does this get used???????
+      end
```

Comment:
If the multi_domain flag is enabled, we find the domain in the list of passed domains, and use this index to update the other variables based on the lists in the end-user's initializer.